pyseir/models/suppression_policies.py

```diff
@@ -157,6 +157,31 @@ def get_interventions(start_date=datetime.now().date()):
     return interp1d(t_list, rho, fill_value='extrapolate')
 
 
+def generate_two_step_policy(t_list, eps, t_break):
```

Comment:
How this is used as opposed to `generate_empirical_distancing_policy`? Is the latter ever used for inference?

Reply:
Since we were having issues with that one, I switched us for right now to use the two step model which fits much better. I think we need to explore data quality issues on that side.